target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6018-cpr-regulator.dtsi

```diff
@@ -1,124 +0,0 @@
-/*
- * Copyright (c) 2019-2020 The Linux Foundation. All rights reserved.
- *
- * Permission to use, copy, modify, and/or distribute this software for any
- * purpose with or without fee is hereby granted, provided that the above
- * copyright notice and this permission notice appear in all copies.
- *
- * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
- * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
- * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
- * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
- * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
- * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
- * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
- */
-
-&soc {
-	apc_apm: apm@b111000 {
-		compatible = "qcom,ipq807x-apm";
-		reg = <0x0 0xb111000 0x0 0x1000>;
-		reg-names = "pm-apcc-glb";
-		qcom,apm-post-halt-delay = <0x2>;
-		qcom,apm-halt-clk-delay = <0x11>;
-		qcom,apm-resume-clk-delay = <0x10>;
-		qcom,apm-sel-switch-delay = <0x01>;
-	};
-
-	apc_cpr: cpr4-ctrl@b018000 {
```

Comment:
The router with pmic is working in my test, can you test it?

Reply:
I didn't mean that board would not work at all, only that CPR (Core Power Reduction) feature will stop working. With main branch I get 887mV on CPU supply (@1.8GHz), while with this PR I get 1062mV on the same board. This would unnecessarily increase power consumption.

Commands used for testing:
```
echo 1800000  > /sys/devices/system/cpu/cpufreq/policy0/scaling_min_freq
grep s2 /sys/kernel/debug/regulator/regulator_summary
```